.circleci/config.yml

```diff
@@ -1,73 +1,97 @@
 # Use the latest 2.1 version of CircleCI pipeline process engine. See: https://circleci.com/docs/2.0/configuration-reference
 version: 2.1
+
 jobs:
-  build:
+  build_check_install:
+    parameters:
+      package:
+        type: string
     docker:
       - image: rocker/tidyverse:3.6.2
     steps:
       - checkout:
-          path: rtables
+          path: pkgsource
+      - run:
+          name: "Build R package"
+          command: |
+            R CMD build pkgsource
+            echo "export PKGBUILD=$(echo *.tar.gz)" >> $BASH_ENV
+      - run:
+          name: "Install R package"
+          command: R CMD INSTALL ${PKGBUILD}    
+      - run:
+          name: "Check R package"
+          command: |
+            R CMD check --no-manual ${PKGBUILD}
+            result=$(tail -1 << parameters.package >>.Rcheck/00check.log)
+            [[ $result = Status:* ]] || echo "could not get Status line"; set -o errexit
+            [[ $result = *"Error"* ]] && echo "checks result in errors"; set -o errexit
       - run:
-          name: "build package"
-          command: R CMD build rtables
-      - persist_to_workspace:
-          root: ./
-          paths:
-            - "*.tar.gz"
+         name: "Prepare Artifacts"
+         command: |
+           mkdir artifacts
+           cp ${PKGBUILD} artifacts/
+      - store_artifacts:
+          path: "artifacts"
+          destination: /
+      - store_test_results:
+          path: << parameters.package >>.Rcheck/tests
+
 
-  check:
+  roxygen:
```

Comment:
rd-up-to-date checking should come before R CMD check and end the pipeline if it fails because doing the expensive check is pointless if the checks  aand example running isn't happening on the correct files anyway